dae-alright/src/components/Footer/Footer.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import './Footer.scss'
+
+
+function Footer() {
+    return (
+        <footer className="deyAlrightFooterContainer">
+            <div className="logoDiv">
+             <img src="https://res.cloudinary.com/dq9utkjw2/image/upload/v1572357304/logo_zrlxud.png" alt="company-logo" className="footerLogo" />
```

Comment:
Which company's logo? Alt should be descriptive, that is what the screen reader wants to read to a visually impaired person. Kindly update your alt property appropriately.

Reply:
changed the alt attribute from company-logo to Dae-Alright Logo